react/README.md

```diff
@@ -279,6 +279,22 @@
   <div />
   ```
 
+- Avoid using array index as `key` prop. ([why?](https://medium.com/@robinpokorny/index-as-a-key-is-an-anti-pattern-e0349aece318))
+
+```jsx
```

Comment:
I don't think the indentation is related to the `>` blockquote. I think it is just the codeblock needs to be indented.